### ISSUE TITLE
multiple vrapper buttons at eclipse toolbar

### ISSUE THREAD

**Issue by galrogo** [vrapper/vrapper#323]
Shortly after I installed Vrapper for eclispe with a few plugins the vrapper activation button on the toolbar started to replicate. First there were 2 buttons and now there's 3. I tried to remove Vrapper and then to reinstall it, but the 3 buttons popped back up immeidately.

**Comment by Angel-O** [comment]
having the exact same issue with jeeeyul's theme icon in the toolbar: it's 4 of them now. Please reopen the issue.

**Comment by albertdev** [comment]
@Angel-O We have never been able to reproduce the issue but have found multiple reports of people having this with random plugins.

Can you check if you Eclipse is installed in C:\Program Files\ or some other path with a space in it?

Have you tried to pick a new Workspace folder in the Preferences screen? Make sure to export only your preferences and keybinds, the window layout should not be copied to the fresh workspace.

**Comment by adelarsq** [comment]
@albertdev @Angel-O I don't think that is a Vrapper bug because the same problem occurs with others plugins. If I remember correctly the only way to solve it was to create another Eclipse workspace.

**Comment by Angel-O** [comment]
@albertdev I am on a mac environment. Eclipse is installed in the application folder. I have noticed though that launching the application from there opens up Eclipse Mars, (I'm on Neon).
The Neon app is in another folder: at the root in the user folder. 
@adelarsq I tried to switch to a new workspace: haven't seen the problem occurring, but that means losing a lot of customisation done in the past weeks and moving projects around I guess...
